tools/plugins/com.liferay.ide.upgrade.core/src/com/liferay/blade/upgrade/liferay70/BREAKING_CHANGES.markdown

```diff
@@ -4341,4 +4341,36 @@ You should change the package name via "correct automatically" in liferay-ide up
 
 #### Why was this change made? [](id=why-was-this-change-made-104)
 
-To leverage the benefits of modularization in Liferay 7.
\ No newline at end of file
+To leverage the benefits of modularization in Liferay 7.
```

Comment:
@codyhoag  FYI we are adding some more BreakingChanges that we have found while testing COde Upgrade Tool.  Is there already some official doc around this breaking change?

Reply:
@gamerson there's the official doc outlined above ([Classes Moved from portal-service.jar](https://dev.liferay.com/develop/reference/-/knowledge_base/7-0/classes-moved-from-portal-service-jar)) and a generic article for [configuring dependencies](https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/configuring-dependencies). Is that what you're looking for?